samples/core/guide/autograph.ipynb

```diff
@@ -620,9 +868,7 @@
         "\n",
         "Since writing control flow in AutoGraph is easy, running a training loop in a TensorFlow graph should also be easy.  \n",
         "\n",
-        "<!--TODO(markdaoust) link to examples showing autograph **in** keras models when ready-->\n",
-        "\n",
-        "Important: While this example wraps a `tf.keras.Model` using AutoGraph, `tf.contrib.autograph` is compatible with `tf.keras` and can be used in [Keras custom layers and models](https://tensorflow.org/guide/keras#build_advanced_models). The easiest way is to `@autograph.convert()` the `call` method.\n",
+        "Important: While this example wraps a `tf.keras.Model` using AutoGraph, `tf.contrib.autograph` is compatible with `tf.keras` and can be used in [Keras custom layers and models](https://tensorflow.org/guide/keras#build_advanced_models). \n",
```

Comment:
Use a DevSite callout, options are:
Note:, Caution:, Warning:, Key Point:, etc

Reply:
!

I was sure "important" was one of the allowed ones, maybe that was in g3doc....

Fixed/reworded.